src/SFA.DAS.EAS.Portal.Worker/EventHandlers/ProviderRelationships/AddedAccountProviderEventHandler.cs

```diff
@@ -1,74 +1,26 @@
-using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Extensions.Logging;
+using SFA.DAS.EAS.Portal.Application.EventHandlers;
 using SFA.DAS.EAS.Portal.Application.Services;
-using SFA.DAS.EAS.Portal.Worker.TypesExtensions;
 using SFA.DAS.ProviderRelationships.Messages.Events;
-using SFA.DAS.Providers.Api.Client;
 
 namespace SFA.DAS.EAS.Portal.Worker.EventHandlers.ProviderRelationships
 {
-    //todo: subscribe to provider updated event (is there 1?) and update our local store provider details
-    
-    /// <remarks>
```

Comment:
Do you believe these notes provide no value, or do you not believe in documentation in general?

I can understand the removal of the section on 'Options for retrieving provider details' somewhat, as it pertains to a decision made, as opposed to providing context to the existing code. However, the 'Notes on handling provider already exists' does provide context to the code in the method (now moved into the Handler), that you can't get from just reading the code.

Reply:
If it's a TODO either do it or add a tech debt item.  If you need excessive comments to understand the code then there is a problem with the code.  Notes on general design decisions like the "Notes on handling provider already exists" would be better in a readme file.  Looking at the code the notes offer no value to this unit of code.  The unit tests should provide that value.

Reply:
Personally, I value lowering the bar for developers new to the code base, over saving space at the top of the file, that can be easily ignored by developers already familiar with the system.